src/svg-renderer.js

```diff
@@ -58,6 +59,13 @@ class SvgRenderer {
         return [this._measurements.width, this._measurements.height];
     }
 
+    /**
+     * @return {Array<number>} the bounds of the canvas this SVG was rendered onto.
```

Comment:
I think the comment here should explicitly mention that it's scaled by the stage scale, so the difference between `size` and `renderBounds` is more clear from outside this class.

Also, "bounds" to me suggests that the return value will be `[left, right, width, height]` or `[left, right, top, bottom]` or something like that -- I'd suggest replacing "bounds" with "size" or "dimensions" or something else that implies magnitude without position.